Octokit.Reactive/Clients/IObservableTeamsClient.cs

```diff
@@ -124,16 +124,52 @@ public interface IObservableTeamsClient
         /// <returns>Newly created <see cref="Team"/></returns>
         IObservable<Team> Create(string org, NewTeam team);
 
+        /// <summary>
+        /// Updates a team
+        /// To edit a team, the authenticated user must either be an organization owner or a team maintainer
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28#update-a-team">API documentation</a> 
+        /// for more information.
+        /// </remarks>
+        /// <returns>updated <see cref="Team" /> for the current org</returns>
+        IObservable<Team> Update(string org, string teamSlug, UpdateTeam team);
+
         /// <summary>
         /// Returns updated <see cref="Team" /> for the current org.
+        /// This endpoint route is deprecated and will be removed from the Teams API.
+        /// We recommend migrating your existing code to use the new Update a team endpoint
         /// </summary>
+        /// <remarks>
+        /// See the <a href="https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28#update-a-team-legacy">API documentation</a> 
+        /// for more information.
+        /// </remarks>
         /// <exception cref="ApiException">Thrown when a general API error occurs.</exception>
         /// <returns>Updated <see cref="Team"/></returns>
         IObservable<Team> Update(int id, UpdateTeam team);
 
         /// <summary>
-        /// Delete a team - must have owner permissions to this
+        /// To delete a team, the authenticated user must be an organization owner or team maintainer.
+        /// If you are an organization owner, deleting a parent team will delete all of its child teams as well.
         /// </summary>
+        /// <remarks>
+        /// See the <a href="https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28#delete-a-team">API documentation</a> 
+        /// </remarks>
+        /// <param name="org">The organization name. The name is not case sensitive.</param>
+        /// <param name="teamSlug">The slug of the team name.</param>
+        /// <exception cref="ApiException">Thrown when a general API error occurs.</exception>
+        /// <returns></returns>
+        IObservable<Unit> Delete(string org, string teamSlug);
+
+        /// <summary>
+        /// Delete a team - must have owner permissions to do this
+        /// This endpoint route is deprecated and will be removed from the Teams API.
+        /// We recommend migrating your existing code to use the new Delete a team endpoint.
```

Comment:
What are your thoughts on providing a link or some other manner to direct users towards the new endpoint?

Reply:
That is a great suggestion. I will provide a link to the new endpoint in the comments.

Reply:
Added /// <see cref="Delete(string, string)"/>.